app/Models/CategoryDAO.php

```diff
@@ -435,13 +436,12 @@ public static function countUnread(array $categories, int $minPriority = 0): int
 		return $n;
 	}
 
-	public static function daoToCategoryPrepopulated($listDAO) {
+	/**
+	 * @param array<string,mixed> $listDAO
+	 * @return array<int,FreshRSS_Category>
+	 */
+	private static function daoToCategoryPrepopulated(array $listDAO) {
```

Comment:
Native array return type forgotten: fixed in https://github.com/FreshRSS/FreshRSS/pull/6046